litmus-portal/frontend/src/views/Analytics/ApplicationDashboards/Cards/CardContent.tsx

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Can remove eslint disable from here as well

Reply:
This page will be refactored it will be updated after this beta.